src/Sulu/Bundle/ContentBundle/Preview/PreviewRenderer.php

```diff
@@ -71,12 +71,14 @@ public function __construct(
      */
     public function render(PageBridge $content, $partial = false)
     {
+        $currentRequest = $this->requestStack->getCurrentRequest();
+
         // set active theme
         $webspace = $this->webspaceManager->findWebspaceByKey($content->getWebspaceKey());
         $this->activeTheme->setName($webspace->getTheme()->getKey());
 
         // get controller and invoke action
-        $request = new Request();
+        $request = new Request($currentRequest->query->all(), $currentRequest->request->all(), [], $currentRequest->cookies->all());
```

Comment:
What does this change have to do with the media provider? And why don't we use `clone`?